src/views/CampusSafety/views/LostAndFound/views/MissingItemCreate/index.tsx

```diff
@@ -286,68 +252,69 @@ const MissingItemFormCreate = () => {
                   </FormGroup>
                 </Grid>
               </Grid>
-              <Grid item margin={2}>
-                <TextField
-                  fullWidth
-                  multiline
-                  minRows={4}
-                  variant="filled"
-                  label={'Location Lost: Be as detailed as possible (or "unknown")'}
-                  name="locationLost"
-                  value={formData.locationLost}
-                  onChange={handleChange}
-                  error={!!validationErrors.locationLost}
-                  helperText={validationErrors.locationLost}
-                />
-              </Grid>
+            </Grid>
+            <Grid item sm={5} xs={12}>
+              {/* Item Brand and Description */}
               <Grid item margin={2}>
                 <TextField
                   fullWidth
                   variant="filled"
-                  label={'Date Lost'}
-                  InputLabelProps={{ shrink: true }} //Shrink label to fit above date placeholder
-                  name="dateLost"
-                  type="date"
-                  value={formData.dateLost}
+                  label={'Item Brand or Make'}
+                  name="brand"
+                  value={formData.brand}
                   onChange={handleChange}
+                  error={!!validationErrors.brand}
+                  helperText={validationErrors.brand}
                 />
               </Grid>
               <Grid item margin={2}>
                 <TextField
                   fullWidth
+                  multiline
+                  minRows={5}
                   variant="filled"
-                  label={'Phone Number'}
-                  name="phoneNumber"
-                  value={formData.phoneNumber}
+                  label={'Item Description: Be as detailed as possible'}
+                  name="description"
+                  value={formData.description}
                   onChange={handleChange}
-                  error={!!validationErrors.phoneNumber}
-                  helperText={validationErrors.phoneNumber}
+                  error={!!validationErrors.description}
+                  helperText={validationErrors.description}
                 />
               </Grid>
+
+              {/* Location Lost and Date Lost */}
               <Grid item margin={2}>
                 <TextField
                   fullWidth
+                  multiline
+                  minRows={4}
```

Comment:
minRows={8} to fill right column of form

Reply:
why 8? seems fine on my end, is there a specific reason?